src/util-profiling.c

```diff
@@ -87,6 +87,9 @@ SCProfilePacketData packet_profile_app_pd_data6[257];
 SCProfilePacketData packet_profile_detect_data4[PROF_DETECT_SIZE][257];
 SCProfilePacketData packet_profile_detect_data6[PROF_DETECT_SIZE][257];
 
+SCProfilePacketData packet_profile_log_data4[LOGGER_SIZE][257];
+SCProfilePacketData packet_profile_log_data6[LOGGER_SIZE][257];
```

Comment:
It looks like the update func doesn't handle the tunnel packets separately, so then the size can be 256 instead.